integration/templates/chart_operator_values.go

```diff
@@ -5,7 +5,7 @@ package templates
 // ChartOperatorValues values required by chart-operator-chart.
 const ChartOperatorValues = `
 clusterDNSIP: 10.96.0.10
-e2e: true
+e2e: false
```

Comment:
This is for the giantswarm-critical priority class. Since this is still on Helm 2 its created by EnsureTillerInstalled.